libs/designsystem/src/lib/components/progress-circle/progress-circle-ring.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, ChangeDetectionStrategy, HostBinding } from '@angular/core';
+
+import { ThemeColor } from '../../helpers/theme-color.type';
+
+const STROKE_WIDTH = 4;
+
+@Component({
+  selector: 'kirby-progress-circle-ring',
+  templateUrl: './progress-circle-ring.component.svg',
+  styleUrls: ['./progress-circle-ring.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProgressCircleRingComponent {
+  @Input() radius: number; // The desired outer radius of the SVG circle
+  @Input() value: number = 0;
+  @Input() themeColor: ThemeColor;
+
+  get offset(): number {
+    return this.centerCircumference - this.centerCircumference * (this.value / 100);
+  }
+
+  @HostBinding('style.width.px')
+  @HostBinding('style.height.px')
+  get diameter(): number {
+    return this.radius * 2;
+  }
+
+  get centerRadius(): number {
+    return this.radius - this.strokeWidth / 2;
+  }
+
+  get centerCircumference(): number {
+    return this.centerRadius * 2 * Math.PI;
+  }
+
+  get strokeWidth(): number {
+    return STROKE_WIDTH;
+  }
```

Comment:
Consider replacing with variable:
```suggestion
  readonly strokeWidth = 4;
```

Reply:
Changed to readonly.